_events/2020-09-19-devs-roundtable.md

```diff
@@ -0,0 +1,19 @@
+---
+site: 
+tags:
+title: "Galaxy Developer Roundtable"
+starts: 2020-09-19
+ends: 2020-09-19
+organiser:
+location: online
+supporters:
+---
+
+Like every other Thursday at **6 pm CET**, the Galaxy Developer Roundtable will take place. Today, [**John Chilton**](https://galaxyproject.org/people/john-chilton/) is the presenter. 
```

Comment:
Is that not every second Thursday...?

Reply:
means the same, left? @simonbray we need a native English speaker here ;)

Reply:
Let me rephrase, is "Like every second Thursday" not better as we do not know when people will read it?

Reply:
changed! :)

Reply:
Sorry, I missed this. Every second, every other, either is fine.